### PR TITLE
week 6 paper draft update

### PR DESCRIPTION
**Pull request by AndrewRqy**
update the most recent draft on Knowledge Distillation and ICRefine

### CODE REVIEW

**Comment by harvey-fin** [comment]
The paper is in good shape. Presentation-wise I think it is helpful to add a figure 1 to the paper that provides an overview. In addition, since the variance is high, running the experiments on more seeds would add to statistical significance of the findings.

**Comment by AndrewRqy** [comment]
> The paper is in good shape. Presentation-wise I think it is helpful to add a figure 1 to the paper that provides an overview. In addition, since the variance is high, running the experiments on more seeds would add to statistical significance of the findings.

Thanks a lot for the feedback! here is the 3-seed mean result for 8 tasks in total for the ICR_holistic v7 pipeline. 
holistic v7 results
Noted that the regression on SN is anticipated since CS-ICL initialized with near perfect training &testing accuracy for that task, so any further refinement on the remaining few errors likely leads to overfitting on the training data
